### ISSUE TITLE
find by district is not showing all the vaccination centres !

### ISSUE THREAD

**Issue by froster02** [cowinapi/developer.cowin#245]
while filtering via state and district the website is not showing all the vaccination centre which are available in the district but can only be seen using the particular pin code of an area.

**Comment by amanjain325** [comment]
@froster02 findbydistrict api doesn't seem to have cached data, it provides real-time data. I use the same.

**Comment by LovishPrajapati** [comment]
I faced the same issue. Not getting full data.

**Comment by kichappa** [comment]
`findByDistrict` returns only available sessions. I am not sure why it has been implemented so. You may use `calendarByDistrict` to get an exhaustive list.

**Comment by amanjain325** [comment]
But it is not real-time.

> `findByDistrict` returns only available sessions. I am not sure why it has been implemented so. You may use `calendarByDistrict` to get an exhaustive list.

**Comment by kichappa** [comment]
> But it is not real-time.

Of course. Not getting real-time data is a bummer. You can try implementing SMS [forwarders] to get real-time data using the protected APIs.
